ecrpc/external_coordinator.swagger.json

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
Q: Is this automatically generated ?

Reply:
Yes, It is generated automatically when we run `buf generate` in the build:
https://github.com/mohamedawnallah/Distributed-Mission-Control-for-LND/blob/2a6eea81db8badc02df5535b7999cf34d04e4a7d/buf.gen.yaml#L16-L17